packages/ERTP/globals.d.ts

```diff
@@ -1,2 +1,10 @@
-declare var makeKind: function;
-declare var makeVirtualScalarWeakMap: function;
+interface VatData {
```

Comment:
I don't know how to do it, but I imagine these types want to live under SwingSet/ and get imported into ERTP, rather than being defined in a file under ERTP/ , for maintenance purposes at the very least. @erights or @michaelfig do you have any idea how to pull that off?

For this importing package's unit tests, the test programs will need to import some sort of `@agoric/swingset/tools/something.js` to get them shimmed into the global, in case that helps anything.

Reply:
I think we should distinguish the type defs that describe the signatures of the various store functions, which probably want to be part of the `store` package, from the declarations that define what globals SwingSet is making available, which arguably want to be part of the `SwingSet` package.  However, the exact nature of the TypeScript voodoo to achieve this eludes me.

There are various of these `globals.d.ts` files scattered throughout, and nearly all of them have the same content.  This is arguably Bad, but I was striving for compatibility with what was already there so I just updated them to correspond to the new reality rather than restructuring how the definitions themselves are packaged.  I think such restructuring ought to happen, though I'm not sure at this point if it should be part of this PR or not.

Reply:
I completely misunderstood this morning. The redundancy has nothing to do with small vs Big stores. I am also mostly ignorant of the typescript foo needed. But this kind of duplication is a maintenance hazard and would be introduced by this PR

@michaelfig @mhofman advice?

Reply:
> But this kind of duplication is a maintenance hazard and would be introduced by this PR

This duplication is not introduced by this PR.  This PR merely perpetuates the badness.